src/Tizen.NUI/src/public/Animation.cs

```diff
@@ -451,6 +459,8 @@ public string[] Properties
         /// <summary>
         /// Gets or sets the destination value for each property of the animation.
         /// </summary>
+        [Obsolete("Deprecated in API9, Will be removed in API11, Please use DestValueList instead")]
```

Comment:
Hello, 
I can see this is for Deprecated-ACR. 
please follow the steps below.
1) add comments like : `//ToDo : will raise deprecated-ACR, [Obsolete("Deprecated in API9, Will be removed in API11. Please use DestValueList instead!")]`
2) get review in DevelNUI with **No "ACR Required" tag**
3) when this is merged in master branch by integration work, you can modify the comment of the `[Obsolete(...)]` tag to **be activated** and use it for ACR process (need to submit new PR in master branch)
  (example) `//ToDo : will raise deprecated-ACR, [Obsolete("...")]` => `[Obsolete("...")]`
4) the PR for ACR should be reviewed on **master branch only**
thank you.